Source/NSImage.m

```diff
@@ -1967,15 +1967,24 @@ - (void) encodeWithCoder: (NSCoder*)coder
 - (id) initWithCoder: (NSCoder*)coder
 {
   BOOL flag;
-
+  NSImage *replacementImage;
```

Comment:
Side remark, I'm not aware of a compiler in use in 2017 that would mind if `replacementImage` and `imageName` were declared closer to their actual place of use.

Reply:
I admit to not liking the 'locals at the top' convention even while I choose to follow it.